plugins/git/git.plugin.zsh

```diff
@@ -349,6 +349,7 @@ alias grhh='git reset --hard'
 alias grhk='git reset --keep'
 alias grhs='git reset --soft'
 alias gpristine='git reset --hard && git clean --force -dfx'
+alias gwipe='git reset --hard && git clean --force -df'
```

Comment:
I wasn't sure if the alias were supposed to be sorted alphabetically or not, so I added it next to gpristine, since they're similar.